variables.tf

```diff
@@ -791,7 +791,7 @@ variable "indico_devops_aws_region" {
 
 variable "thanos_enabled" {
   type    = bool
-  default = false
+  default = true
```

Comment:
ditto here for removing the variable

Reply:
I intentionally left all the code commented out, so that it would be easy to turn this feature back on in the future. So, I was thinking that it would be better to keep the variable, rather than removing it.